src/Dropdown/Dropdown.tsx

```diff
@@ -63,6 +64,10 @@ export const Dropdown = forwardRef(function Dropdown(
   }: DropdownProps,
   ref: ForwardedRef<HTMLSelectElement>,
 ) {
+  const [value, setValue] = useControllableState({
+    initialState: defaultValue,
+    stateProp: valueProp,
+  })
```

Comment:
`Dropdown` behaviour when used is still the same
Internally tho, it's always controlled even when only `defaultValue` is provided